Makefile.am

```diff
@@ -91,3 +91,8 @@ dist_sbin_SCRIPTS = \
 	eos-repartition-mbr \
 	eos-update-flatpak-repos \
 	$(NULL)
+
+tmpfilesddir = $(prefix)/lib/tmpfiles.d
```

Comment:
(Really minor) nitpick: This variable is called `tmpfilesdir` in `systemd.pc` (note `dir` not `ddir`), so we should probably name it consistently with that to ease greppability.

Reply:
Fair point. Fixed.